addons/im_livechat_mail_bot/models/mail_bot.py

```diff
@@ -12,15 +12,15 @@ def _get_answer(self, record, body, values, command):
         if self._is_bot_in_private_channel(record):
             if odoobot_state == "onboarding_attachement" and values.get("attachment_ids"):
                 self.env["mail.canned.response"].create({
-                    "source": "Thanks",
-                    "substitution": "Thanks for your feedback. Goodbye!",
-                    "description": "This is a temporary canned response to see how canned responses work.",
+                    "source": _("Thanks"),
+                    "substitution": _("Thanks for your feedback. Goodbye!"),
+                    "description": _("This is a temporary canned response to see how canned responses work."),
                 })
                 self.env.user.odoobot_failed = False
                 self.env.user.odoobot_state = "onboarding_canned"
                 return Markup(_("Wonderful! 😇<br/>Try typing %s to use canned responses. I've created a temporary one for you.", "<span class=\"o_odoobot_command\">:</span>"))
             elif odoobot_state == "onboarding_canned" and self.env.context.get("canned_response_ids"):
-                self.env["mail.canned.response"].search([
+                self.env["mail.canned.response"].with_context(lang=None).search([
                     ("create_uid", "=", self.env.user.id),
                     ("source", "=", "Thanks"),
                     ("description", "=", "This is a temporary canned response to see how canned responses work."),
```

Comment:
Is it really necessary to filter on `description`? Isn't `source` enough?
The `unlink()` might fail in case the string is worded differently. `Thanks` could also be changed but this is much less likely to happen.